lib/collection.js

```diff
@@ -2052,6 +2052,9 @@ Collection.prototype.mapReduce = function(map, reduce, options, callback) {
  */
 Collection.prototype.initializeUnorderedBulkOp = function(options) {
   options = options || {};
+  if (this.s.options.ignoreUndefined) {
+    options.ignoreUndefined = this.s.options.ignoreUndefined;
```

Comment:
@kvwalker should this method have `ignoreUndefined` supported as an `option`?

Reply:
`initializeUnorderedBulkOp` and `initializeOrderedBulkOp` should both support `ignoreUndefined`, since this option is applied here: https://github.com/mongodb/node-mongodb-native/blob/next/lib/bulk/common.js#L983. This is definitely something I missed in options management so I'll go back and verify all of those options. Thanks for catching this, @amotzte! Could you also add support for `ignoreUndefined` to `initializeOrderedBulkOp` and add documentation for both?

Reply:
We should make sure that if a user specifies `ignoreUndefined` on `options`, we do not overwrite it with `this.s.options.ignoreUndefined`